### PR TITLE
Adjust shapes params

### PR DESCRIPTION
**Pull request by melonora**
Same as #267 but then for shapes.

### CODE REVIEW

File: src/spatialdata_plot/pl/basic.py [hunk 1]
```diff
@@ -164,18 +166,18 @@ def render_shapes(
...
         **kwargs: Any,
     ) -> sd.SpatialData:
         """
         Render shapes elements in SpatialData.
 
         Parameters
         ----------
-        elements : list[str] | str | None, optional
+        element : list[str] | str | None, optional
             The name(s) of the shapes element(s) to render. If `None`, all shapes
```
Comment (Sonja-Stockhaus) [inline src/spatialdata_plot/pl/basic.py:178]: Doesn't apply anymore. In general, the docstring needs to be updated in several places, right?

Comment (melonora) [reply src/spatialdata_plot/pl/basic.py:178]: yes, I will adjust the docstrings if overall the PR is accepted:) Just the type annotations already had to be changed for mypy

Comment (Sonja-Stockhaus) [reply src/spatialdata_plot/pl/basic.py:178]: Ah okay, perfect

**Review by Sonja-Stockhaus — APPROVED** [review]

**Comment by melonora** [comment]
@Sonja-Stockhaus cheers for reviewing! I will adjust the docstrings after finishing PR in the spatialdata repo

**Pull request merged**